subsys/net/lib/lwm2m/lwm2m_shell.c

```diff
@@ -57,7 +57,7 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
 static void send_cb(enum lwm2m_send_status status)
 {
-	LOG_INF("SEND status: %d\n", status);
+	LOG_INF("SEND status: %d", status);
```

Comment:
As there are logger changes, perhaps change the commit subject and body to
```
net: lwm2m: removes redundant newline for log and shell calls

All the shell and logger calls here already add a newline.
```

Not all shell printing macros add `\n`, but all shell/log calls in this file do that so can omit `\n`

Reply:
> nitpicky

yes, very much so, sorry about that